connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -108,15 +108,17 @@ message Join {
   }
 }
 
-// Relation of type [[Union]], at least one input must be set.
-message Union {
+// Relation of type [[SetOperation]]
+message SetOperation {
   repeated Relation inputs = 1;
```

Comment:
shall we require 2 inputs `left` and `right`? Our SQL parser only generates set operation with 2 children.

Reply:
I see
```
queryTerm
    : queryPrimary                                                                       #queryTermDefault
    | left=queryTerm {legacy_setops_precedence_enabled}?
        operator=(INTERSECT | UNION | EXCEPT | SETMINUS) setQuantifier? right=queryTerm  #setOperation
    | left=queryTerm {!legacy_setops_precedence_enabled}?
        operator=INTERSECT setQuantifier? right=queryTerm                                #setOperation
    | left=queryTerm {!legacy_setops_precedence_enabled}?
        operator=(UNION | EXCEPT | SETMINUS) setQuantifier? right=queryTerm              #setOperation
```

Reply:
Done. Now the definition is easier to follow: just check if there is left or right.